scheduler/src/cook/mesos/util.clj

```diff
@@ -64,7 +65,7 @@
   (lookup-cache! cache :db/id miss-fn entity))
 
 (defonce ^Cache job-ent->resources-cache (new-cache))
-(defonce ^Cache categorize-job-cache (new-cache))
+(defonce ^Cache job-ent->pool-cache (new-cache))
```

Comment:
Is it better to set up this cache as `job-uuid->pool` ?

Reply:
I'm not sure. My goal here was to keep this as close to the previous job to category cache as possible.

Reply:
okay.